src/OpenLoco/Map/Map.hpp

```diff
@@ -27,14 +27,14 @@ namespace OpenLoco::Map
     static_assert(sizeof(Pos3) == 6);
     static_assert(sizeof(TilePos2) == 4);
 
-    constexpr bool validCoord(coord_t coord)
+    constexpr bool validCoord(const coord_t& coord)
```

Comment:
technically a `coord_t` is faster to pass by value than by reference. This is the opposite of a Pos2 or TilePos2 which are faster to pass by reference.